projects/exercises/01/CMakeLists.txt

```diff
@@ -0,0 +1,3 @@
+add_executable(drive_in_line drive_in_line.cpp)
```

Comment:
No CMake on the robots this year. The IDE already has a "Runner" script in it that pulls in all of the libraries needed.